public/app/core/components/AppChrome/NavToolbar/NavToolbarSeparator.tsx

```diff
@@ -30,6 +30,9 @@ const getStyles = (theme: GrafanaTheme2) => {
       height: 24,
       flexShrink: 0,
       flexGrow: 0,
+      [theme.breakpoints.down('sm')]: {
+        display: 'none',
+      },
```

Comment:
I'm wondering if this should be the case for all `NavToolbarSeparators`. Here is an example where I think separators still make sense:

https://github.com/user-attachments/assets/29ef4cca-f738-4d17-a8ff-933ddbb3462f

Reply:
@svennergr looks good to me, on mobile they don't add much and take up precious space 

![Screenshot 2025-04-08 at 10 05 44](https://github.com/user-attachments/assets/cb3c7c08-ccaf-471a-92f1-0805353d7248)